cpp/src/reductions/scan/scan_inclusive.cu

```diff
@@ -178,17 +183,17 @@ struct scan_dispatcher {
    */
   template <typename T, typename std::enable_if_t<is_supported<T>()>* = nullptr>
   std::unique_ptr<column> operator()(column_view const& input,
-                                     null_policy null_handling,
+                                     null_policy,
                                      rmm::cuda_stream_view stream,
                                      rmm::mr::device_memory_resource* mr)
   {
-    return inclusive_scan<T>(input, null_handling, stream, mr);
+    return scan_functor<Op, T>{}.inclusive_scan(input, stream, mr);
```

Comment:
If you make `inclusive_scan` static and rename to `invoke`, then you'd have the "dispatch to invoke" pattern ;) 
```suggestion
    return scan_functor<Op, T>::invoke(input, stream, mr);
```